app/services/api/jsonrpc/jsonrpc.ts

```diff
@@ -78,23 +68,21 @@ export class JsonrpcService extends Service implements IJsonrpcServiceApi {
     data: any;
     isRejected?: boolean;
   }): IJsonRpcResponse<IJsonRpcEvent> {
-    return this.createResponse<IJsonRpcEvent>(null, {
+    return this.createResponse<IJsonRpcEvent>(undefined, {
       _type: 'EVENT',
       ...options,
     });
   }
 
   createError(
-    requestOrRequestId: string | IJsonRpcRequest,
     options: { code: E_JSON_RPC_ERROR; message?: string },
+    request?: IJsonRpcRequest,
   ): IJsonRpcResponse<any> {
-    // eslint-disable-next-line
-    return JsonrpcService.createError.apply(this, arguments);
+    return JsonrpcService.createError.apply(this, [options, request]);
   }
 
   createRequest(resourceId: string, method: string, ...args: any[]): IJsonRpcRequest {
-    // eslint-disable-next-line
-    return JsonrpcService.createRequest.apply(this, arguments);
+    return JsonrpcService.createRequest.apply(this, [resourceId, method, args]);
```

Comment:
@andycreeth I assume changes of this type caused this behavior. Probably it needed to be `...args` instead of `args` to unwrap the content of `args` in the apply function?

Reply:
Let me know if I should make a patch for this. :)

Reply:
Yup that was the issue.  I opted to revert the PR rather than debugging in the moment, but I agree that seems like the fix.  If you prepare a new PR we can go through a more extensive set of tests this time.

Reply:
No problem and sorry for any troubles I might have caused. I will prepare these days a PR and double check and test the changes too.